jung-visualization/src/main/java/edu/uci/ics/jung/visualization/util/LayoutMediator.java

```diff
@@ -0,0 +1,89 @@
+package edu.uci.ics.jung.visualization.util;
+
+import com.google.common.graph.Network;
+import edu.uci.ics.jung.algorithms.layout.Layout;
+import edu.uci.ics.jung.algorithms.util.IterativeContext;
+import java.awt.*;
+import java.awt.geom.Point2D;
+import java.util.Set;
+import java.util.function.Function;
+
+/** Created by Tom Nelson on 9/22/17. */
```

Comment:
Please remove this comment and all other "Created by..." comments like this.

Reply:
Is someone planning to remove all of the author names in comments for the 'legacy' code?

Reply:
Good question. :)

Not yet, but I've added it as an item to #101, so it's tracked now.

Reply:
There's a couple of reasons. Firstly, I wanted the author(s) removed because I automatically associate comments of this style with auto-generated comments from IDEs, which I personally find unhelpful. Secondly, I just simply hadn't realised before writing my initial comment above that comments like this do already exist in the code base.

I personally advocate either removing author comments of this style, or alternatively rejigging them so they have the following format (which I believe is more standard in the Java ecosystem). But if you'd rather keep author comments as they are for consistency, I'd totally understand.

```java
/**
 * (Class javadoc goes here.)
 *
 * @author Tom Nelson
 */
```